client/src/components/Sub_ProductDetail/ImageGallery.jsx

```diff
@@ -1,11 +1,12 @@
 import React, { useState, useEffect } from 'react';
-import ImageSlide from './Sub_ImageGallery/ImageSlide.jsx';
-import DefaultView from './Sub_ImageGallery/DefaultView.jsx';
+import ModalView from './Sub_ImageGallery/ModalView.jsx';
 
 function ImageGallery({ currentStyle, productID }) {
-  console.log('SHOW CURRENT STYLE: ', currentStyle);
+  // console.log('SHOW CURRENT STYLE: ', currentStyle);
   const [currentImageIndex, setCurrentImageIndex] = useState(0);
   const [styleMemo, setStyleMemo] = useState({});
+  const [showModalView, setShowModalView] = useState('default');
```

Comment:
I think if we're going to have a modular modal and we're passing in different `view` states, I think we should leave a comment above line 8 to specify the potential options that `showModalView` can be to increase code readability.
ex:
```
/** 
 * @param {('default' | "expand-view" | zoom')} initialState 
*/
```

Reply:
I also believe a more fitting name for your state on line 8 would be `modalView` instead of `showModalView`. `showModalView` sounds to me like a name for a state handler function rather than the state itself.

Reply:
nice catch!

Reply:
good point